assets/airports/efhk.json

```diff
@@ -0,0 +1,1532 @@
+        "PEPIG": ["N60d08m29", "E22d35m59"],
+        "PEXEN": ["N59d54m47.00", "E23d49m28.00"],
+        "PISOX": ["N59d46m41", "E22d45m04"],
+        "PODOM": ["N60d12m20.89", "E24d40m10.57"],
+        "POGOK": ["N59d59m28", "E021d23m16"],
+        "RATLA": ["N60d51m30", "E28d20m18"],
+        "RATMU": ["N60d14m16", "E26d32m09"],
+        "REDBO": ["N60d25m38.00", "E24d51m52.00"],
+        "REKDO": ["N59d26m35", "E021d33m40"],
+        "RENKU": ["N59d53m09.00", "E24d53m48.10"],
+        "RILBO": ["N60d11m37.00", "E24d16m43.00"],
+        "RIPVI": ["N60d16m24.00", "E24d35m07.00"],
+        "RIRIP": ["N60d12m10", "E021d26m03"],
+        "ROKVI": ["N60d51m39", "E27d01m23"],
+        "ROLAT": ["N60d17m57.00", "E25d31m20.00"],
+        "ROPAM": ["N60d07m59.00", "E25d36m06.00"],
+        "RUMEP": ["N59d30m45", "E22d25m56"],
+        "SORPA": ["N59d37m01", "E24d50m49"],
+        "SUTAX": ["N60d29m38.26", "E24d59m05.08"],
+        "TEDMU": ["N60d27m46.64", "E25d14m09.63"],
+        "TEVRU": ["N60d49m16.00", "E24d49m29.00"],
+        "TIXED": ["N60d25m44.59", "E25d11m49.01"],
+        "TLL"  : ["N59d24m41", "E24d50m18"],
+        "TN425": ["N59d48m10.3", "E24d53m11.0"],
+        "TN471": ["N59d34m15.4", "E25d02m47.4"],
+        "TN903": ["N59d37m11.0", "E24d26m19.1"],
+        "TUSBI": ["N61d04m05", "E26d18m23"],
+        "UMUGI": ["N60d02m14", "E22d41m41"],
+        "UNURU": ["N60d27m53.00", "E24d45m56.00"],
+        "UREDA": ["N60d14m20.00", "E24d37m23.00"],
+        "USITU": ["N59d51m31", "E021d21m57"],
+        "USUPO": ["N61d36m45", "E025d00m56"],
+        "UXADA": ["N60d51m00", "E022d36m48"],
+        "VAGIP": ["N60d18m35.00", "E24d25m20.00"],
+        "VAMRA": ["N60d13m33.00", "E24d33m27.00"],
+        "VAVIS": ["N60d14m06.00", "E24d50m39.00"],
+        "VEPIN": ["N60d37m53.00", "E26d19m59.00"],
+        "VETUD": ["N60d30m30.00", "E24d48m41.00"],
+        "VIBEP": ["N60d27m33.00", "E24d14m39.00"],
+        "XELBU": ["N60d23m36.37", "E25d04m53.30"],
+        "XOMRI": ["N60d26m34.00", "E25d13m38.00"]
+    },
+    "runways": [
+        {
+            "name": ["04L", "22R"],
+            "name_offset": [[0.9, 1], [0, 0]],
+            "end": [
+                ["N60d18m46.61", "E24d54m13.93", "133.6ft"],
+                ["N60d19m53.42", "E24d56m40.90", "179.9ft"]
+            ],
+            "ils": [true, true]
+        },
+        {
+            "name": ["04R", "22L"],
+            "name_offset": [[0.9, 1], [-0.8, -1]],
+            "end": [
+                ["N60d18m40.65", "E24d56m10.94", "153.2ft"],
+                ["N60d19m50.49", "E24d58m44.73", "148.6ft"]
+            ],
+            "ils": [true, true]
+        },
+        {
+            "name": ["15", "33"],
+            "name_offset": [[0, 0], [-0.7, 1]],
+            "end": [
+                ["N60d19m48.99", "E24d57m52.19", "162.9ft"],
+                ["N60d18m25.44", "E24d59m17.83", "147.1ft"]
+            ],
+            "ils": [true, false]
+        }
+    ],
+    "sids": {
```

Comment:
The format has also changed;

Here's an example of the new format:
```
"MASBO4B": {
    "icao": "MASBO4B",
    "name": "Masbo Four Bravo",
    "rwy": {
        "WSSS20C": []
    },
    "body": [["IBIXU", "A15+"], ["IBIVA", "A25+"], "DONDI", ["DIVSA", "A40+|S230"], "DOSPA", ["VTK", "A70+"], "AKOMA", ["AGVAR", "A110+"], "SABKA"],
    "exitPoints": {
        "MASBO": ["MASBO"]
    },
    "draw": [
        ["MASBO"]
    ]
},
```

Most notably, we don't use `suffix`es anymore; each procedure has its own object. ie. There would be an individual entry for the SID MASBO2E and an individual entry for MASBO4B at Singapore. Also, the `rwy` section **must** list all runways the procedure is valid for, and must also contain the airport ICAO ie. `WSSS02C` instead of `02C`

Reply:
Great. That seemed to me the only approach sustainable in long-term when I encountered some issues (not with EFHK but with that Swedish airport (Lulea?)) where same runway had more than one procedure for the same runway and waypoint combination (the other not meant for all aircraft types).